app/components/Nav.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Link, NavLink } from 'react-router-dom';
+
+const views = ['Home', 'Campuses', 'Students']
+
+export default class Nav extends Component {
+  render() {
+    return (
+      // <div>
+      <nav className="navbar navbar-default">
+        <NavLink to="/">
+          <span className="navbar-brand" >Margaret Hamilton Academy</span>
+        </NavLink>
+        <div>
+          <ul className="nav nav-pills">
+            {views.map(view => {
+              let path;
+              if (view === 'Home') {
+                path = '/'
+              } else {
+                path = `/${view}`
+              }
+              return (
+                <NavLink className="nav nav-pills" to={`${path}`}>
+                  <li key={`${view}`} className="navBarList" ><span>{view}</span></li>
```

Comment:
I think in this case spreading the links across a loop makes this a little harder to read. I think the repetition of code isn't too bad in this case:

```jsx
<NavLink className="nav nav-pills" to="/Home">
  <li key="Home" className="navBarList"><span>Home</span></li>
</NavLink>

<NavLink className="nav nav-pills" to="/Campuses">
  <li key="Campuses" className="navBarList"><span>Campuses</span></li>
</NavLink>

<NavLink className="nav nav-pills" to="/Students">
  <li key="Students" className="navBarList"><span>Students</span></li>
</NavLink>
```

Or even better, this might be a good spot for a new sub-component:

```jsx
function SiteNavLink ({viewName, path})  {
  return (
    <NavLink className="nav nav-pills" to={path}>
      <li key={viewName} className="navBarList"><span>{viewName}</span></li>
    </NavLink>
  );
}

<SiteNavLink path='/' viewName='Home'/>
<SiteNavLink path='/Campuses' viewName='Campuses'/>
<SiteNavLink path='/Students' viewName='Students'/>
```

But the combination of looping over the ray and the conditional logic in the `if clause` made me work a little harder to understand this component.